src/frontend/src/common/utils/featureFlags.tsx

```diff
@@ -25,6 +25,10 @@ export const FEATURE_FLAGS: FlagsObj = {
     isDisabled: false,
     key: "editTrees",
   },
+  editSamples:{
```

Comment:
ah, now I see what the linter is complaining about. space after colon

Reply:
lol i thought i could get away with not linting since i way only adding like 3 lines 😆